demo/pages/index.js

```diff
@@ -4,29 +4,18 @@ const Header = dynamic(() => import(/* webpackChunkName: 'header' */ '../compone
 import { useRouter } from 'next/router'
```

Comment:
These are just some updates to the demo index page, as it used deprecated syntax